cinema/views.py

```diff
@@ -43,6 +48,21 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+        actors = self.request.query_params.get("actors")
+        genres = self.request.query_params.get("genres")
+        title = self.request.query_params.get("title")
+        if actors:
+            actors_id = [int(str_id) for str_id in actors.split(",")]
```

Comment:
Ensure that the conversion of actor IDs to integers in `get_queryset` handles potential ValueErrors gracefully, especially if the input is not a valid integer.